javascript/javascript3/redo/week1/weatherApp.js

```diff
@@ -0,0 +1,30 @@
+document.querySelector(".submitCityButton").addEventListener("click", () => fetchWeatherData())
+const weatherDataDiv = document.querySelector(".weatherDataDiv")
+
+function fetchWeatherData() {
+    const city = document.querySelector(".city").value
+    if (!city) { city = "copenhagen" }
+
+    fetch(`https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=85bf3bafaf784a301c66129107680db7`)
+        .then(response => response.json())
+        .then(weatherData => {
+            weatherDataDiv.innerHTML = `
```

Comment:
Just curious, do you feel comfortable with the DOM API as well?
Using innerHTML with template strings feels very mustache like so it's easier to write out but it's good to also understand the DOM API.

Reply:
Thanks for mentioning. I read more about it.